### DISCUSSION TITLE
anyone know how to play sound file from resource/raw folder in android?

### DISCUSSION THREAD

**Discussion by yoer** [bluefireteam/audioplayers#1646] [General]
### Checklist

- [X] I read the [troubleshooting guide](https://github.com/bluefireteam/audioplayers/blob/main/troubleshooting.md) before raising this issue
- [X] I made sure that the issue I am raising doesn't already exist

### Current bug behaviour

``` dart
DeviceFileSource("resource://raw/notification")
``` 
i try to use **DeviceFileSource**, but i get these errors:
``` 
AudioPlayers Exception: AudioPlayerException(
I/flutter (14045): 	DeviceFileSource(path: resource://raw/notification.mp3), 
I/flutter (14045): 	PlatformException(MEDIA_ERROR_UNKNOWN {what:1}, MEDIA_ERROR_SYSTEM, null, null)
```

file:
![image](https://github.com/bluefireteam/audioplayers/assets/11307627/3d15f22b-dcc4-4842-8705-f8893b6b27a5)

### Expected behaviour

play normal

### Steps to reproduce

1. Execute `flutter run` on the code sample 
2. ... 
3. ... 

### Code sample

<details>
<summary>Code sample</summary>

```dart
DeviceFileSource("resource://raw/notification")
```

</details>

### Affected platforms

Android

### AudioPlayers Version

main

### Build mode

debug

### Logs

```
my relevant logs
```

<details>
  <summary>Full Logs</summary>

  

  ```
  my full logs or a link to a gist
  ```

  Flutter doctor:
  ```
  Output of: flutter doctor -v
  ```
</details>

### Working on PR

no way

**Comment by yoer** [comment]
anyone know? i stuck

**Comment by Gustl22** [comment]
Flutter is expected to be used on multiple platforms, not just Android. So move your resources to the `assets` folder in the root directory and use `AssetSource('notification.mp3')` instead.

**Comment by yoer** [comment]
thanks
